playbooks/roles/check_duplicate_irlm/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+#
+# Copyright 2024 IBM Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Check if IRLM is already up
+- name: Check for duplicate IRLM
+  collections:
+      - ibm.ibm_zos_core
+      - ibm.ibm_zos_ims
+  block:
+    - name: Query IRLM subsystem
+      ibm.ibm_zos_core.zos_operator:
+        cmd: 'D SSI,SUB={{ DXR_IRLM_SSID }}'
+      register: irlm_output
+
+    - name: Check if IRLM subsystem exists 
+      ansible.builtin.set_fact:
+        irlm_is_up: true
+      when:
+        - irlm_output.content is defined
+        - '"THERE IS NO INFORMATION TO DISPLAY" not in irlm_output.content | join'
+
+    - name: 'Check if an IRLM subsystem with SSID {{ DXR_IRLM_SSID }} already exists. Trying to reconcile...'
```

Comment:
Drop the second sentence in the task name because it's a conditional task therefore not always true.